step-templates/swaggerhub-post-api.json

```diff
@@ -0,0 +1,117 @@
+{
+    "Id": "0f60f13c-bc00-4035-a0ba-5aeef6a2c1e5",
+    "Name": "SwaggerHub Post Api",
+    "Description": "Post your Swagger/OAS to SwaggerHub",
```

Comment:
Please add an "Author" parameter and include your GitHub username.

Reply:
Done